docs/uncertainty/index.rst

```diff
@@ -1,10 +1,10 @@
 .. _astropy-uncertainty:
 
-.. testsetup::
-
-    >>> import numpy as np
-    >>> np.random.seed(12345)
-
+.. testsetup::

```

Comment:
I didn't do this intentionally (I'm guessing my IDE did).
GitHub apparently doesn't render this change in a useful manner but diffing in a terminal reveals what's going on:
```
diff --git a/docs/uncertainty/index.rst b/docs/uncertainty/index.rst
index cf1151e9bc..7ca2dc8568 100644
--- a/docs/uncertainty/index.rst
+++ b/docs/uncertainty/index.rst
@@ -1,10 +1,10 @@
 .. _astropy-uncertainty:
 
-.. testsetup::
-
-    >>> import numpy as np
-    >>> np.random.seed(12345)
-
+.. testsetup::^M
+^M
+    >>> import numpy as np^M
+    >>> np.random.seed(12345)^M
+^M
```
`^M` being a carriage return character.
I could try to revert this part of the diff since it's unrelated and distracting, though I suspect this could pop up again next time I or someone else tries to edit this file, so maybe it's better to bite the bullet now ?

Reply:
I prefer we revert if we can. Some devs are bound to use editor in Windows. We cannot sanitize everything, especially not in this PR. Thank you for your understanding.

Reply:
The operation was a bit tricky but I think I got it.

Reply:
> We cannot sanitize everything, especially not in this PR.

The Ruff formatter is already preventing mixed line endings in Python files and we could (and should!) use [a pre-commit hook](https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#mixed-line-ending) for the rest. But I agree that it is beyond the scope of this pull request.

Reply:
So, is the problem that this file has ^M^J everywhere else, and the IDE just did it for those lines too? I guess we need a clean-up operation...

Anyway, the best surely for this PR is just not to have those changes - thanks for undoing them!